frontend/src/metabase/timelines/common/containers/EventForm/EventForm.tsx

```diff
@@ -0,0 +1,14 @@
+import { connect } from "react-redux";
+import { State } from "metabase-types/store";
+import EventForm, {
+  EventFormOwnProps,
+  EventFormStateProps,
+} from "../../components/EventForm";
+
+const mapStateToProps = (state: State) => ({
+  formattingSettings: state.settings.values["custom-formatting"],
```

Comment:
Think we should use the [`getSetting` selector](https://github.com/metabase/metabase/blob/53effa4a5e44529e49b6cba936054248d2c5e3fe/frontend/src/metabase/selectors/settings.js#L5) here?

Reply:
Fixed, didn't know about it